frontend/src/components/my-items/AddItemForm.jsx

```diff
@@ -78,7 +103,14 @@ export default function AddItemForm (props) {
         </Form.Group>
         <Button variant="outline-primary" type="submit" className="me-1 button"
           onClick={handleSubmit}>Submit</Button>
-        <Button variant="outline-danger" type="reset" className="button">Reset</Button>
+        <Button variant="outline-danger" type="reset" className="button" onClick={handleReset}>Reset</Button>
+        {showAlert
+          ? (
+            <Alert variant="warning" onClose={() => setShowAlert(false)} dismissible>
+            Please enter a {missingField}
```

Comment:
This leads to typos because of a/an, e.g. "Please enter a item". Could do something like "Missing a required field: {missingField}" instead.